frontend/src/components/modalContent/AddPartyModal.js

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { getFormValues, submit, reset } from "redux-form";
+import { Steps, Button, Popconfirm } from "antd";
+import * as FORM from "@/constants/forms";
+import CustomPropTypes from "@/customPropTypes";
+import { createParty } from "@/actionCreators/partiesActionCreator";
+import AddFullPartyForm from "@/components/Forms/parties/AddFullPartyForm";
+
+const { Step } = Steps;
+
+const propTypes = {
+  fetchData: PropTypes.func.isRequired,
+  createParty: PropTypes.func.isRequired,
+  submit: PropTypes.func.isRequired,
+  reset: PropTypes.func.isRequired,
+  title: PropTypes.string,
+  addPartyFormValues: PropTypes.objectOf(PropTypes.strings),
+  addPartyForm: PropTypes.objectOf(CustomPropTypes.genericFormState),
+};
+
+const defaultProps = {
+  addPartyFormValues: {},
+  addPartyForm: {},
+  title: "",
+};
+
+export class AddPartyModal extends Component {
+  state = { isPerson: true, current: 0 };
+
+  togglePartyChange = (value) => {
+    this.setState({ isPerson: value.target.value });
+  };
+
+  handlePartySubmit = (event, addAnother) => {
+    event.preventDefault();
+    const type = this.state.isPerson ? "PER" : "ORG";
+    const payload = { type, ...this.props.addPartyFormValues };
+    return this.props
+      .createParty(payload)
+      .then(() => {
+        this.props.reset(FORM.ADD_FULL_PARTY);
+        if (addAnother) {
+          this.prev();
+        } else {
+          this.props.fetchData();
+          this.props.closeModal();
+        }
+      })
+      .catch(() => {
+        this.prev();
+      });
+  };
+
+  next() {
+    if (!this.props.addPartyForm.syncErrors) {
+      const current = this.state.current + 1;
+      this.setState({ current });
+    } else {
+      // submit form to trigger validation errors.... alternate suggestions??
+      this.props.submit(FORM.ADD_FULL_PARTY);
+    }
+  }
+
+  prev() {
+    const current = this.state.current - 1;
+    this.setState({ current });
+  }
+
+  renderStepOne() {
+    return (
+      <AddFullPartyForm
+        {...this.props}
+        onSubmit={this.handlePartySubmit}
+        isPerson={this.state.isPerson}
+        togglePartyChange={this.togglePartyChange}
+        handleFormReset={this.handleFormReset}
+      />
+    );
+  }
+
+  // WIP
+  renderStepTwo() {
+    return (
+      <div className="center">
+        <p>
+          If you would like to add another contact, click on the add another contact below. Your
```

Comment:
`click on the add another contact ____ below`. There should probably be something in that blank. Looks like `button` might make sense

Reply:
fixed!